onduty/redacted.env

```diff
@@ -2,5 +2,5 @@
 GITHUB_ACCESS_TOKEN=[REDACTED]
 GITHUB_ORG=ampproject
 RELEASE_ONDUTY_TEAM=release-on-duty
-BUILD_COP_TEAM=build-cop
+BUILD_ON_DUTY_TEAM=build-on-duty
```

Comment:
ONDUTY/ON_DUTY >_<

Reply:
Nice catch. Fixed.

Reply:
I would suggest the opposite, replace `ONDUTY` with `ON_DUTY`. Also, don't forget to change it in both the env file and in the code!

Reply:
Agree. Done, and cross-checked.